build/jobs/run-export-tests.yml

```diff
@@ -171,7 +177,7 @@ jobs:
     displayName: 'Export E2E ${{ parameters.version }} SQL'
     inputs:
       command: test
-      arguments: '"**\*${{ parameters.version }}.Tests.E2E*.dll" --filter "FullyQualifiedName~SqlServer&Category=ExportLongRunning"'
+      arguments: '"$(Agent.TempDirectory)/E2ETests/**/*${{ parameters.version }}.Tests.E2E*.dll" --filter "FullyQualifiedName~SqlServer&Category=ExportLongRunning"'
       workingDirectory: "$(System.ArtifactsDirectory)"
```

Comment:
Should the parameter name used as argument also contain "Microsoft.Health.Fhir."?

Reply:
It could, but matching "R4.Tests.E2E.dll" also is pretty specific, in this PR I've only corrected the path to `$(Agent.TempDirectory)/E2ETests`. 
After messing around with the build files recently I'm wanting to do a larger scale cleanup at some point too :)